scalingo/data_scalingo_container_size.go

```diff
@@ -0,0 +1,103 @@
+package scalingo
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/Scalingo/go-scalingo/v5"
+)
+
+func dataSourceScContainerSize() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceScContainerSizeRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"sku": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"human_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"human_cpu": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"memory": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"pids_limit": {
```

Comment:
Can you open an issue on the dev doc as this attribute is missing (https://developers.scalingo.com/container_sizes)?

Reply:
https://github.com/Scalingo/developers-documentation/issues/238